source/includes/administration/_identity_providers.md

```diff
@@ -228,6 +235,8 @@ Required | &nbsp;
 `logo`<br/>*string* | A base64 encoded data url or url to an image for the logo to display on the login screen. If of a default provider type, this will be set with a default if not passed.
 `type`<br/>*string* | The type of authentication protocol. Possible values: OIDC, SAML.
 `parameters`<br/>*Array* | A list of parameters associated with the identity provider. The issuerURL will be set if of a default provider type.
+`rank`<br/>*int* | If provided, it sorts the of identity providers in decending order starting from 1.
```

Comment:
_If provided, sorts the identity providers on the login page in ascending order._